ab_tool/management/commands/unassigned_students.py

```diff
@@ -0,0 +1,26 @@
+from django.core.management.base import BaseCommand, CommandError
+
+from ab_tool.canvas import get_unassigned_students_with_stored_credentials
+from ab_tool.exceptions import NoValidCredentials
+from ab_tool.models import Course
+
+class Command(BaseCommand):
+    args = ""
+    help = "Sends email alerts about courses with unassigned students"
+    
+    def handle(self, *args, **options):
+        # TODO: update notification messages
+        for course in Course.get_notification_courses():
```

Comment:
So, basically you want to loop over all experiments that are finalized and have csv_upload as the track weighting.  Courses/experiments should also have an end date, so that would further limit the number we're talking about (we can add that in later).  I'd rather see a single loop at the experiment level where you check to see if the associated course is notifiable than have an inner loop.  I also think we can have a better name than `experiments_to_check` and that it can be a method present on the Experiment object - perhaps `Experiment.get_all_finalized_csv` or something that's more reflective of what the method is doing.